src/testshade/optixgridrender.cpp

```diff
@@ -272,15 +272,15 @@ OptixGridRenderer::synch_attributes()
         const ustring* cpuString
             = (const ustring*)(colorSys
                                + (cpuDataSize
-                                  - sizeof(StringParam) * numStrings));
+                                  - sizeof(ustringhash) * numStrings));
         CUdeviceptr gpuStrings = d_color_system + podDataSize;
         for (const ustring* end = cpuString + numStrings; cpuString < end;
              ++cpuString) {
             // convert the ustring to a device string
             uint64_t devStr = cpuString->hash();
```

Comment:
OptiX tests are crashing here with an invalid `cpuString` pointer.

Reply:
@steenax86 , as this is the "cpuString" side I things, the
`sizeof(StringParam)`
would map to a ustring vs. ustringhash.
We see later on line 280 the hashcode is extracted from it:
`uint64_t devStr = cpuString->hash();`

So try changing it to 
`sizeof(ustring)`